tests/common/unittest_common.cpp

```diff
@@ -760,6 +760,18 @@ TEST (conf_custom, env_str_01)
 int
 main (int argc, char **argv)
 {
-  testing::InitGoogleTest (&argc, argv);
-  return RUN_ALL_TESTS ();
+  int ret = 0;
+  try {
+    testing::InitGoogleTest (&argc, argv);
+  } catch (...) {
+    g_warning ("catch 'testing::internal::<unnamed>::ClassUniqueToAlwaysTrue'");
```

Comment:
Is `...ClassUniqueToAlwaysTrue` the unique exception that can be thrown during `testing::InitGoogleTest()`?

Reply:
Yes.
Also this exception is never thrown actually - https://github.com/google/googletest/blob/master/googletest/src/gtest.cc#L5711